### ISSUE TITLE
Release CRAN version 1.5.0

### ISSUE THREAD

**Issue by alexzwanenburg** [alexzwanenburg/familiar#85]
**Prior to merge with master**
- [x] increment package version
- [x] create release name
- [x] update NEWS.md
- [x] code check: run `devtools::check(args=c("--no-examples", "--no-tests"), vignettes=FALSE)`, alternatively `rcmdcheck::rcmdcheck(args=c("--no-examples", "--no-tests"))`
- [x] test check: run `devtools::test()`: set `options("testthat.progress.max_fails"=Inf)` and `options("Ncpus"=10)`
- [x] create archives for testing backward compatibility against future versions: run `tests/old_experiments/archiveExperiments.R`
- [x] test backward compatibility against archived experiments: manually run `tests/testthat/test-update_object.R`
- [x] pre-compile vignettes: run `vignettes/compile.R`
- [x] code and vignette check: `rcmdcheck::rcmdcheck(args=c("--no-examples", "--as-cran"))`
- [x] check reverse dependencies
- [x] create pull request
- [x] check continuous integration tests
- [x] Build source tarball using `devtools::build()`.
- [x] Run `devtools::check_built(path=pkg, env_vars = list("_R_CHECK_DEPENDS_ONLY_" = TRUE))`, with pkg the path to the tarball.
- [x] update cran_comments.md
- [x] merge dev branch into master

**Post-merge**
- [x] Check for merging errors.
  - [x] Run a post-merge code check: `rcmdcheck::rcmdcheck(args=c("--no-examples", "--as-cran"))`
  - [x] Run post-merge test suite, if required.
  - [x] Pre-compile vignettes, if required.
  - [x] Rebuild source tarball using `devtools::build()`, if required.

**CRAN**
- [x] Check CRAN-policies on https://cran.r-project.org/web/packages/policies.html
- [x] Upload source tarball to https://cran.r-project.org/submit.html
- [x] Check CRAN checks

**Github release**
- [x] prepare a GithHub release
  - [x] copy news from NEWS.md
  - [x] add source tarball as data

**Comment by alexzwanenburg** [comment]
Version name: Whole Whale

**Comment by alexzwanenburg** [comment]
The only issue directly tied to `familiar` is the following:
```
✖ | 1 1    406 | 0_plot_calibration [5480.6s]                                                                                                                                                                                                              

Warning (test-0_plot_calibration.R:40:1): 3. Plots for multinomial outcomes can be created for a dataset with some missing data and a naive model.
'fun.aggregate' is NULL, but found duplicate row/column combinations, so defaulting to length(). That is, the variables [ensemble_model_name, positive_class, data_set, fs_method, learner, type] used in 'formula' do not uniquely identify rows in the input 'data'. In such cases, 'fun.aggregate' is used to derive a single representative value for each combination in the output data.table, for example by summing or averaging (fun.aggregate=sum or fun.aggregate=mean, respectively). Check the resulting table for values larger than 1 to see which combinations were not unique. See ?dcast.data.table for more details.
Backtrace:
    ▆
 1. ├─base::do.call(...) at familiar/R/TestFunctions.R:4828:9
 2. ├─familiar (local) `<nnstndGF>`(object = `<fmlrCllc>`)
 3. └─familiar (local) `<nnstndGF>`(object = `<fmlrCllc>`) at familiar/R/PlotCalibration.R:147:5
 4.   └─familiar:::.plot_calibration_plot(...) at familiar/R/PlotCalibration.R:595:7
 5.     └─familiar:::.create_calibration_plot(...) at familiar/R/PlotCalibration.R:734:5
 6.       ├─data.table::dcast(data = linear_test, cast_formula, value.var = "value") at familiar/R/PlotCalibration.R:873:5
 7.       └─data.table::dcast.data.table(...)
 8.         └─data.table:::warningf(...)
 9.           └─data.table:::raise_condition(...)

Error (test-0_plot_calibration.R:40:1): 3. Plots for multinomial outcomes can be created for a dataset with some missing data and a naive model.
Error in ``[.data.table`(plot_layout_table, , `:=`(figure_id = .I, is_present = sapply(grobs, 
    gtable::is.gtable)))`: Supplied 7 items to be assigned to 6 items of column 'is_present'. If you wish to 'recycle' the RHS please use rep() to make this intent clear to readers of your code.
Backtrace:
    ▆
 1. ├─base::do.call(...) at familiar/R/TestFunctions.R:4828:9
 2. ├─familiar (local) `<nnstndGF>`(object = `<fmlrCllc>`)
 3. └─familiar (local) `<nnstndGF>`(object = `<fmlrCllc>`) at familiar/R/PlotCalibration.R:147:5
 4.   └─familiar:::.plot_calibration_plot(...) at familiar/R/PlotCalibration.R:595:7
 5.     └─familiar:::.update_plot_layout_table(...) at familiar/R/PlotCalibration.R:812:3
 6.       ├─...[] at familiar/R/PlotUtilities.R:989:3
 7.       └─data.table:::`[.data.table`(...) at familiar/R/PlotUtilities.R:989:3
```

I established that this was an issue with the testing code (and logic). This is fixed with version 2.0.0.
